crates/rpc/src/types/query.rs

```diff
@@ -4,3 +4,8 @@ use serde::Deserialize;
 pub struct RandaoQuery {
     pub epoch: Option<u64>,
 }
+
+#[derive(Debug, Deserialize)]
+pub struct ValidatorBalanceQuery {
+    pub id: Option<Vec<String>>,
```

Comment:
A enum should be used instead of a string here

Reply:
Fixed in the latest comment with `ValidatorID` type